src/api/mod.rs

```diff
@@ -1390,6 +1413,71 @@ impl<'a> AuthenticatedApi<'a> {
     }
 }
 
+/// Available datasets for fetching organization events
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Dataset {
+    /// Our logs dataset
+    OurLogs,
+}
+
+impl Dataset {
+    /// Returns the string representation of the dataset
+    pub fn as_str(&self) -> &'static str {
+        match self {
+            Dataset::OurLogs => "ourlogs",
+        }
+    }
+}
```

Comment:
I'd add a `Display` implementation here. Then, you can just print the Dataset enum directly, without having to call `as_str` on it.
```suggestion
}

impl fmt::Display for Dataset {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        write!(f, "{}", self.as_str())
    }
}

```